tf-modules/report-pdrs/README.md

```diff
@@ -0,0 +1,26 @@
+# Report PDRs
+
+This package includes a Lambda function to process PDR ingest notifications received via SNS and store PDR data to a database.
+
+## Deployment
+
+1. Copy the .tfvars sample file: `cp terraform.tfvars.sample terraform.tfvars`
+2. Populate the sample file with values that apply to your AWS environment (see configuration variables section below).
+3. Deploy this module: `terraform apply`
+
+NOTE: Terraform will ignore the `aws_profile` config variable if you have static credentials or environment variables set, see the [AWS Provider page](https://www.terraform.io/docs/providers/aws/index.html#authentication).
```

Comment:
I think perhaps we should refactor our modules to remove the `aws_profile` var. It's kind of unnecessary because I expect most people are using the profile in their environment, and it would allow us to remove one of our config `var`s, which is always a good thing, and these doc notes that are not related to the module. It's not even really something useful for checking into version control because everybody has a different local profile setup if they use profiles at all.